libraries/interop-ribs/src/main/kotlin/com/bumble/appyx/interop/ribs/InteropActivity.kt

```diff
@@ -17,8 +17,10 @@ abstract class InteropActivity : RibActivity(), IntegrationPointProvider {
         )
 
     override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
+        // super.onCreate() creates RIB with AppyxNode inside. It's important to have
+        // appyxIntegrationPoint ready before we create a root node
         appyxIntegrationPoint = createAppyxIntegrationPoint(savedInstanceState)
+        super.onCreate(savedInstanceState)
```

Comment:
> If someone creates their own `InteropActivity` (because they cannot extend this), they will likely have the same problem. Should we add this 'gotcha' to the documentation?

I'm not sure it's gonna be a problem. If they don't use this, they will have to create integration point manually, gaining full control over it

Reply:
Gaining full control to implement it incorrectly though?

Reply:
They just need to follow the same order as in `RIBs`: 
1. create integration point
2. create root node

But we can mention it explicitly